akka-persistence-tck/src/main/scala/akka/persistence/journal/JournalSpec.scala

```diff
@@ -181,8 +183,11 @@ abstract class JournalSpec(config: Config) extends PluginSpec(config) {
       journal ! ReplayMessages(0, Long.MaxValue, Long.MaxValue, pid, receiverProbe.ref)
       receiverProbe.expectMsg(RecoverySuccess(highestSequenceNr = 5L))
     }
+  }
 
-    "reject non-serializable events" in {
+  "A Journal" may {
+    "reject non-serializable events" in optional("This journal does not reject non-serializable events. This is legal.") {
+      1 should ===(2)
```

Comment:
Ah yeah, will remove ofc; this was used to generate: https://github.com/akka/akka/pull/19011#issuecomment-159589805